CMakeLists.txt

```diff
@@ -17,8 +18,17 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/drishti-upload/cmake/Mo
 # drishti specific hunter cache pacakges:
 list(APPEND HUNTER_CACHE_SERVERS "https://github.com/elucideye/hunter-cache")
 
-include(drishti_set_hunter_gate)
-drishti_set_hunter_version(DRISHTI_HUNTER_GATE_URL DRISHTI_HUNTER_GATE_SHA1)
+if(EXISTS "${drishti_upload_modules}")
+  include(drishti_set_hunter_gate)
+  drishti_set_hunter_version(DRISHTI_HUNTER_GATE_URL DRISHTI_HUNTER_GATE_SHA1)
+else()
+  # Release archive will not contain submodule 'drishti-upload'.
```

Comment:
Workaround for https://github.com/headupinclouds/drishti_hunter_test/issues/1

Reply:
👍